deploy/examples/cluster-test.yaml

```diff
@@ -57,6 +59,23 @@ spec:
     mgr: system-cluster-critical
   disruptionManagement:
     managePodBudgets: true
+
+
+resources:
```

Comment:
The resources need to be indented two spaces, so they will be at the same indentation as other settings.

Reply:
@travisn  wrote:
> The resources need to be indented two spaces, so they will be at the same indentation as other settings.

Thanks for the hint! It allowed me to deploy a cluster again!

Just to make this perfectly clear: I did not add this modified file to the PR with the intention to get it merged eventually, but just to demonstrate and track what I'm trying to use for local testing ...